server/controllers/caseNotesController.ts

```diff
@@ -58,19 +53,29 @@ export default class CaseNotesController {
       const prisonApiClient = this.prisonApiClientBuilder(clientToken)
       const [caseNotesUsage, caseNotesPageData] = await Promise.all([
         prisonApiClient.getCaseNotesUsage(req.params.prisonerNumber),
-        this.caseNotesService.get({
-          token: clientToken,
-          prisonerData,
-          queryParams,
-          canViewSensitiveCaseNotes: !!permissions.sensitiveCaseNotes?.view,
-          canDeleteSensitiveCaseNotes: !!permissions.sensitiveCaseNotes?.delete,
-          currentUserDetails: res.locals.user,
-        }),
+        Result.wrap(
+          this.caseNotesService.get({
+            token: clientToken,
+            prisonerData,
+            queryParams,
+            canViewSensitiveCaseNotes: !!permissions.sensitiveCaseNotes?.view,
+            canDeleteSensitiveCaseNotes: !!permissions.sensitiveCaseNotes?.delete,
+            currentUserDetails: res.locals.user,
+          }),
+        ),
       ])
 
+      if (caseNotesPageData.status === 'rejected') {
```

Comment:
There's an `isFullfilled()` method on a result, so I think it'd be preferable to do: 

```
if (!caseNotesPageData.isFulfilled()) {
```